app/controllers/api/generic_works_controller.rb

```diff
@@ -0,0 +1,98 @@
+module API
+  # Adds an endpoint that customizes the update action on nested works
+  class GenericWorksController < ApplicationController
```

Comment:
Our approach was to expose a simple add_parent and remove_parent action that responded with JSON, and it seemed like the API was the place to do it. What's a better/appropriate approach to do this sort of thing?